plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -486,6 +510,18 @@ export class PersonState {
 
                 failedAttempts++
                 if (failedAttempts === MAX_FAILED_PERSON_MERGE_ATTEMPTS) {
+                    void captureWarning(
+                        this.jobQueueManager,
+                        teamId,
+                        `Failed to merge "{mergeIntoDistinctId}" and "{otherPersonDistinctId}" too many attempts`,
```

Comment:
The code is not interpolating correctly - $ is missing.

Also we should probably only have a machine-readable token emitted here and have logic to group/display based on that in the FE rather than describing in here - allows us to update documentation going forward.